src/libtcod/zip.hpp

```diff
@@ -159,6 +160,16 @@ public :
 	*/
 	void putConsole(const TCODConsole *val);
 
+	/**
+	@PageName zip_put
+	@FuncTitle Putting a random number generator state in the buffer
+	@Cpp void TCODZip::putRandom(const TCOD_random_t val)
+	@C void TCOD_zip_put_random(TCOD_zip_t zip, const TCOD_random_t val)
+	@Param zip	In the C version, the buffer handler, returned by the constructor.
+	@Param val	An RNG state to store in the buffer
```

Comment:
There was no way for you to know this, but the tags used for most of the current C++ docs are not compatible with the current doc generator.  In the future you could use plain text or Doxygen style tags.